lib/client.js

```diff
@@ -321,11 +339,7 @@ Client.prototype = {
   /// client.trigger('activation') // activating!
   /// ```
   trigger: function(name, data) {
-    if (this._parent) { throw new Error('Not Implemented'); }
-    if (!this._messageHandlers[name]) { return false; }
-    this._messageHandlers[name].forEach(function(handler) {
-      handler(data);
-    });
+    this.postMessage('iframe.trigger:' + name, { message: data });
```

Comment:
do we not want to call `triggerEvent` as well? what about triggering an event in the same instance? (do we have specs for that?)